packages/showcases/src/components/FlowDiagram.tsx

```diff
@@ -17,46 +21,85 @@ const getFlowChartAsset = (showcase: string) => {
     }
 }
 
+const customStyles = {
+    overlay: {
+        background: "black"
```

Comment:
I think the background black gives a feeling that user moved away from the page. Could you check with Kelly if this is okay?